docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to Decentraland's catalyst
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+
+## We Develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
```

Comment:
There are some minor typos in the GitHub naming. `github` / `Github` vs `GitHub`

```suggestion
We use GitHub to host code, to track issues and feature requests, as well as accept pull requests.
```